### PR TITLE
added "Oracle Linux " support

### PR DESCRIPTION
**Pull request by vzell**

### CODE REVIEW

**Comment by hswong3i** [comment]
Wait, adding another platform support is not such simple

1. If tasks/redhat.yml could be shared, don’t add a new one
2. Create vars/oraclelinux-7.yml (!?) for those required platform specific variables
3. (Important !!) create molecule test plan for the new platform, eg LXD based for regular Travis CI tests, or Vagrant based adhoc localhost VirtualBox tests
4. (VERY IMPORTANT!!) someone need to keep monitoring above tests if always goes healthy...

Basically, my clients only asking for Ubuntu 18.04/RHEL 7/openSUSE 15.1, so I have no intention for Oracle Linux support...
